client/components/date-picker/index.jsx

```diff
@@ -88,7 +96,7 @@ class DatePicker extends PureComponent {
 		return eventsInDay;
 	}
 
-	locale() {
+	localeUtils() {
```

Comment:
Nitpicking a little here but I'd name this `getLocaleUtils` or similar. Generally though, the existing method seems a bit weird :/

Reply:
Moved to https://github.com/Automattic/wp-calypso/pull/29981